docs: updated docs to add product table

burnsjared0415 opened the pull request:
<!-- markdownlint-disable first-line-h1 no-inline-html -->

<!--
    In order to have the best experience with our community, we recommend that you read the code of conduct and contributing guidelines before submitting a pull request.
    
    By submitting this pull request, you confirm that you have read, understood, and agreed to the project's code of conduct and contributing guidelines.

    Please use conventional commits to format the title of the pull request and the commit messages.
    For more information, please refer to https://www.conventionalcommits.org.
-->

### Summary

- Added table to the documentation for Aria-related cmdlets.
- Removed documentation and export for `Get-AriaLocal*` cmdlets.

<!--
    Please provide a clear and concise description of the pull request.
-->

### Type

<!--
    Please check the one(s) that applies to this pull request using "x".
-->

- [ ] Bugfix
- [ ] Enhancement or Feature
- [ ] Code Style or Formatting
- [x] Documentation
- [ ] Refactoring
- [ ] Chore
- [ ] Other
        Please describe:

### Breaking Changes?

<!--
    Please check the one that applies to this pull request using "x".
    If this pull request contains a breaking change, please describe the impact and mitigation path.
-->

- [ ] Yes, there are breaking changes.
- [x] No, there are no breaking changes.

### Test and Documentation

<!--
    Please check the one(s) that applies to this pull request using "x".
    For bug fixes and enhancements/features, please ensure that tests and documentation have been completed and provide details.
-->

- [x] Tests have been completed.
- [x] Documentation has been added or updated.

<!--
    Please describe the tests that have been completed and/or the documentation that has been added/updated.
-->

### Issue References

<!--
    Is this related to any GitHub issue(s)? If so, please provide the issue number(s) that are closed or resolved by this pull request.

    For bug fixes and enhancements/features, please ensure that a GitHub issue has been created and provide the issue number(s) here.

    Please use the 'Closes' keyword followed by the a hash and issue number.
    This will link the pull request to the issue(s) and automatically close them when the pull request is merged.

    Example:

    Closes #000
    Closes #001
-->

### Additional Information

<!--
    Please provide any additional information that may be helpful.
-->

burnsjared0415 committed:
docs: updated docs to add product table

- Added table to the documentation for Aria-related cmdlets.
- Removed documentation and export for `Get-AriaLocal*` cmdlets.

Signed-off-by: Jared Burns <jared.burns@broadcom.com>

tenthirtyam reviewed (changes requested):
Add CHANGELOG for 1.8.1 and bump the module version.

tenthirtyam reviewed (changes requested):
Also, update the mkdocs.yml to reflect that the Get- functions are no longer exported.

tenthirtyam committed:
fix: correct json output (#194)

Updated all `Publish-*` cmdlets to correctly output as JSON with using the `-json` parameter.

Signed-off-by: Ryan Johnson <ryan.johnson@broadcom.com>

bhumitra reviewed (dismissed):
LGTM 🚀
